tools/doc/html.js

```diff
@@ -198,7 +198,8 @@ function preprocessElements({ filename }) {
           heading.children = [{
             type: 'text',
             value: file.contents.slice(
-              position.start.offset, position.end.offset),
+              position.start.offset, position.end.offset)
+              .replace(/\\.{1}/g, (match) => match[1]),
```

Comment:
Maybe just:
```js
.replace(/\\(?=.)/g, ''),
```

Reply:
Or, if we need to exclude escaped escapes:
```js
.replace(/\\(?=[^\\])/g, ''),
```

Reply:
jslint doesn't accept `.replace(/\\(?=.)/g, ''),`:

```
/Users/rubys/git/node/tools/doc/html.js
  202:29  error  Unescaped dot character in regular expression  node-core/no-unescaped-regexp-dot
```

I want to **include** escaped escapes.

Finally, lookahead isn't appropriate here.  If we have an escaped escape, I want to emit a single backslash and then to _not_ consider that backslash as an escape character.  Consider four escapes in a row, the desired result would be two backslashes:

```
> "\\\\\\\\".replace(/\\(?=.)/g, '').length
1
> "\\\\\\\\".replace(/\\.{1}/g, (match) => match[1]).length
2
```

Reply:
What about `.replace(/\\(.{1})/g, '$1'),`? Whould it be simpler and faster then function call and string index access?

Reply:
BTW, we can disable linter rule to simplify RegExp, but I am not sure what is more confusing:
```js
.replace(/\\./g, (match) => match[1]), // eslint-disable-line node-core/no-unescaped-regexp-dot
```